strapi/strapi
# chore(sentry): convert to typescript

### What does it do?

Converts the Sentry plugin codebase to TypeScript

### Why is it needed?

TS all the things

### How to test it

- Sign up on Sentry, create a Node project, copy the DSN.
- In a Strapi app, install the sentry plugin, and add the DSN in the `config/plugins.js`. 
- Throw an error somewhere in your application, like in a controller.
- Trigger the error, it should show up in your Sentry dashboard with appropriate metadata

I've done that manually:

![image](https://github.com/strapi/strapi/assets/8087692/3dceaea5-a17e-407d-97db-a515143cac9f)
